Software/Serial_read/py_arduino_serial_GUI.py

```diff
@@ -317,7 +318,8 @@ def get_ino_params(self):
             dict: Dictionary containing parameters.
         """
         params = {
-            "NUMBER_OF_TRIALS": self.num_trials.value(), 
+            "NUMBER_OF_TRIALS": self.num_trials.value(),
+            "IS_TRAINING": self.is_training.value(),
```

Comment:
should be

`"IS_TRAINING": self.is_training.checkState() == Qt.CheckState.Checked,` 

for checkboxes